pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -289,14 +289,35 @@ const (
 	// at the same time. During a failover, it is crucial to ensure that the old
 	// application is removed before creating a new one to avoid duplicate
 	// processing and maintaining state consistency.
+	// Deprecated: The term Immediately conflicts with the meaning of GracePeriodSeconds
+	// and may cause confusion for users, use Directly instead.
+	// This field is still effective in the current API version but will be
+	// removed in the next API version.
```

Comment:
```suggestion
	//
	// Deprecated: The term 'Immediately' may be confusing when used alongside 'GracePeriodSeconds',
	// which specifies that resources are removed after a grace period rather than at once.
	// 'Immediately' is replaced by 'Directly' for clarity. This term remains functional in the current
	// API version for backward compatibility and will be removed when PropagationPolicy advances to alpha2 or beta.
```